contrib/fonttools/showttf.c

```diff
@@ -2484,10 +2484,13 @@ static void gposPairSubTable(FILE *ttf, int which, int stoffset, struct ttfinfo
 	free(ps_offsets);
     } else if ( subformat==2 ) {
 	printf( "\t   Class based kerning (not displayed)\n" );
-	PrintGlyphs( showCoverageTable(ttf,stoffset+coverage,-1),info);
+	glyphs = showCoverageTable(ttf,stoffset+coverage,-1);
+	PrintGlyphs(glyphs,info);
     } else {
 	printf( "\t   !!! unknown sub-table format !!!!\n" );
     }
+    if (glyphs)
+        free(glyphs);
```

Comment:
`if (glyphs)` implies that `glyphs` could be NULL, which makes it appropriate to do that at the top.

Reply:
Of course, if it *can't* be NULL you can just remove the check.